skimage/feature/_canny.py

```diff
@@ -11,11 +11,13 @@
 All rights reserved.
 Original author: Lee Kamentsky
 """
+from __future__ import division, print_function, absolute_import
 
 import numpy as np
 import scipy.ndimage as ndi
-from scipy.ndimage import (gaussian_filter,
-                           generate_binary_structure, binary_erosion, label)
+from scipy.ndimage import (generate_binary_structure, binary_erosion, label)
+
```

Comment:
This and the future imports I usually tweak when I change other things, but am reluctant to really apply throughout the package because it means essentially every prior PR would need a rebase for the price of improved style...